pyvista/utilities/geometric_objects.py

```diff
@@ -1128,3 +1131,197 @@ def Circle(radius=0.5, resolution=100):
     points[:, 1] = radius * np.sin(theta)
     cells = np.array([np.append(np.array([resolution]), np.arange(resolution))])
     return pyvista.wrap(pyvista.PolyData(points, cells))
+
+
+def PlatonicSolid(kind='tetrahedron', radius=1.0, center=(0.0, 0.0, 0.0)):
+    """Create a Platonic solid of a given size.
+
+    Parameters
+    ----------
+    kind : str or int, optional
+        The kind of Platonic solid to create. Either the name of the
+        polyhedron or an integer index:
+
+            * ``'tetrahedron'`` or ``0``
+            * ``'cube'`` or ``1``
+            * ``'octahedron'`` or ``2``
+            * ``'icosahedron'`` or ``3``
+            * ``'dodecahedron'`` or ``4``
+
+    radius : float, optional
+        The radius of the circumscribed sphere for the solid to create.
+
+    center : sequence, optional
+        Three-length sequence defining the center of the solid to create.
+
+    Returns
+    -------
+    pyvista.PolyData
+        One of the five Platonic solids. Cell scalars are defined that
+        assign integer labels to each face (with array name
+        ``"FaceIndex"``).
+
+    Examples
+    --------
+    Create and plot a dodecahedron
```

Comment:
```suggestion
    Create and plot a dodecahedron.
```

Reply:
Well, well, well... how the turntables :P Thanks, fixing these.